NanoServerPackage/NanoServerPackage.psm1

```diff
@@ -453,14 +444,20 @@ function Install-NanoServerPackage
         }
 
         $packagesToBeInstalled = @()
-
+              
         # do a find first, if there are any errors, don't install
         $packagesToBeInstalled += (Find -Name $Name -MinimumVersion $MinimumVersion -MaximumVersion $MaximumVersion -RequiredVersion $RequiredVersion `
             -Culture $Culture -ErrorAction Stop) # -Repository $Repository
-
+      
         if ($packagesToBeInstalled.Count -eq 0)
-        {
-            return
+        { 
+            ThrowError -CallerPSCmdlet $PSCmdlet `
+                -ExceptionName System.Exception `
```

Comment:
fixed

---
In reply to: [81068506](https://github.com/OneGet/NanoServerPackage/pull/5#discussion_r81068506) [](ancestors = 81068506)